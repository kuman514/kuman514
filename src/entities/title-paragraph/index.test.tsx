import { render } from '@testing-library/react';

import TitleAndParagraph from '.';

describe('TitleAndParagraph', () => {
  it('should render as expected', () => {
    const { container } = render(
      <TitleAndParagraph
        title={
          <>
            <span className="text-green-600 first-letter:font-bold">
              Lorem ipsum dolor sit amet
            </span>
            , consectetur adipiscing elit.
          </>
        }
        paragraphProps={[
          {
            children:
              'Duis sodales ipsum odio, eu accumsan dui ullamcorper sit amet. Nulla tristique ornare cursus. Morbi egestas porta facilisis. Praesent ac ultricies nunc, pretium ullamcorper ipsum. Suspendisse placerat imperdiet ipsum, nec condimentum orci accumsan a. Nullam sollicitudin justo in ex ullamcorper posuere. Vestibulum finibus diam eu metus maximus lacinia. Praesent eleifend hendrerit libero quis egestas. Sed vitae venenatis ante.',
            customClassName:
              'first-letter:text-2xl first-letter:text-green-600 first-letter:font-bold',
          },
        ]}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
