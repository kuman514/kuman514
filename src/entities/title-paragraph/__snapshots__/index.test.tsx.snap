// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleAndParagraph should render as expected 1`] = `
<div>
  <h2
    class="text-bold text-4xl undefined"
  >
    <span
      class="text-green-600 first-letter:font-bold"
    >
      Lorem ipsum dolor sit amet
    </span>
    , consectetur adipiscing elit.
  </h2>
  <p
    class="text-base font-normal indent-4 mb-4 first-letter:text-2xl first-letter:text-green-600 first-letter:font-bold"
  >
    Duis sodales ipsum odio, eu accumsan dui ullamcorper sit amet. Nulla tristique ornare cursus. Morbi egestas porta facilisis. Praesent ac ultricies nunc, pretium ullamcorper ipsum. Suspendisse placerat imperdiet ipsum, nec condimentum orci accumsan a. Nullam sollicitudin justo in ex ullamcorper posuere. Vestibulum finibus diam eu metus maximus lacinia. Praesent eleifend hendrerit libero quis egestas. Sed vitae venenatis ante.
  </p>
</div>
`;
